src/main.js

```diff
@@ -3,11 +3,29 @@
  * SPDX-License-Identifier: AGPL-3.0-or-later
  */
 
+import wrap from '@vue/web-component-wrapper'
+import Vue from 'vue'
+import semver from 'semver'
+
 import FlowNotify from './views/FlowNotify.vue'
 
+let customElementId
+// we may only compare first three numbers of the version
+if (semver.gt(window.OC.config.version.split('.', 3).join('.'), '31.0.2')) {
```

Comment:
Without this condition, I get `this.$root.props is undefined` on NC 31 <= 31.0.2 even though the component is not used :unamused: 

otoh there is also no necessity to create it for nothing, so it is not bad to have the check there.

Reply:
I briefly checked for related API changes that we could detect instead but found none.
Not sure which line throws the undefined error - we could also catch that. But this seems most explicit and clean anyway.

Reply:
This only appears when the component is actually not used. Somewhere along the store's `addRule` call (i showed you the strace). My explanation for now is that there is some event (handler) or so being called and since  this is not embedded properly anywhere it stumbles.